time_to_first_response.py

```diff
@@ -49,6 +51,8 @@ def measure_time_to_first_response(
             number=1, sort="created", direction="asc"
         )  # type: ignore
         for comment in comments:
+            if comment.user.login in ignore_users:
+                continue
```

Comment:
I think there may be an additional change that needs to be made here. Since we are only requesting the first comment (line 51) if that first comment is an ignored comment, then we won't search for the next one. Does that make sense?

Reply:
Thank you so much for your thoughtful review and comments!!

I think this part seems to be working well. At line 51, it appears to me that we are not just requesting the first comment, but an iterator for all the comments. (Below image may be helpful for reference)

We are distinguishing the ignored users against this iterator. As we are using `continue` rather than `break`, I believe it is functioning correctly.

<img width="995" alt="Screenshot 2023-08-01 at 3 23 29" src="https://github.com/github/issue-metrics/assets/86868255/2c4e4c6b-7113-4f16-bf63-e398a77d1be5">

ref. https://github3.readthedocs.io/en/latest/api-reference/issues.html#github3.issues.comment

Reply:
Your correct about the iterator but since we set the value of `number` to 1, there is only 1 comment allowed in the iterator. I have increased to correct the issue.